django/cantusdb_project/main_app/views/views.py

```diff
@@ -555,17 +563,19 @@ def change_password(request):
                 form.user.changed_initial_password = True
             user = form.save()
             update_session_auth_hash(request, user)
-            messages.success(request, 'Your password was successfully updated!')
+            messages.success(request, "Your password was successfully updated!")
     else:
         form = PasswordChangeForm(request.user)
         if request.user.changed_initial_password == False:
             messages.warning(
                 request,
-                "The current password was assigned to you by default and is unsecure. Please make sure to change it for security purposes."
+                (
+                    "The current password was assigned to you by default and is unsecure. "
+                    "Please make sure to change it for security purposes."
+                ),
```

Comment:
@dchiller or @lucasmarchd01, please verify that I've done this correctly!

Reply:
Oops....what am I verifying?